v0/destinations/am/transform.js

```diff
@@ -462,23 +473,35 @@ function processTransaction(message) {
 function process(event) {
   const respList = [];
   const { message, destination } = event;
+
   const messageType = message.type.toLowerCase();
   const eventType = message.event ? message.event.toLowerCase() : undefined;
+
   const toSendEvents = [];
+  let temp;
   if (
     messageType === EventType.TRACK &&
     (eventType === Event.PRODUCT_LIST_VIEWED.name ||
-      eventType === Event.PRODUCT_LIST_CLICKED)
+      eventType === Event.PRODUCT_LIST_CLICKED.name)
   ) {
-    toSendEvents.push(processProductListAction(message));
+    // toSendEvents.push(processProductListAction(message));
+    temp = processProductListAction(message);
+    // eslint-disable-next-line no-plusplus
```

Comment:
don't disable these.. change i++ to I += 1 . 
for others so check what the es lint is saying search and try to replace.